mysql-async/src/test/resources/update-config.sh

```diff
@@ -2,7 +2,8 @@
 cp -f /docker-entrypoint-initdb.d/ca.pem /var/lib/mysql/ca.pem
 cp -f /docker-entrypoint-initdb.d/server-cert.pem /var/lib/mysql/server-cert.pem
 cp -f /docker-entrypoint-initdb.d/server-key.pem /var/lib/mysql/server-key.pem
-cat /etc/mysql/my.cnf
+cat /etc/my.cnf || true
+cat /etc/mysql/my.cnf || true
 ls -lah /var/lib/mysql
-ls -lah /etc/mysql/conf.d/
-ls -lah /etc/mysql/mysql.conf.d/
+ls -lah /etc/mysql/conf.d/ || true
```

Comment:
why was that added?

Reply:
The location of configuration files is different between images for 5.7 and 8.0, so I suppress failures with this.